### DISCUSSION TITLE
Access blocked: This app’s request is invalid with Chicago Crime Analysis - Level 1 test notebook

### DISCUSSION THREAD

**Discussion by rockofut17** [DevSkillsHQ/discusssions#16] [Help] [answered]
Hi, I am the getting the following output when trying to authorize the notebook app for the created project in GCP

![imagen](https://user-images.githubusercontent.com/86205229/228654426-c9b19e34-9259-40b3-a91a-487bfee94efa.png)

This occurs whenever I visit the URL provided when running the copy of the test notebook

![imagen](https://user-images.githubusercontent.com/86205229/228654855-cfec98a7-032d-4e6f-b576-45baf93255db.png)

**Comment by fenske** [comment, marked as answer]
Hi @rockofut17. I'm Anton from DevSkills.

Sorry for keeping you waiting.

This issue is apparently related to some recent general changes that have been done to how Google Colab communicates with BigQuery. And unfortunately, it looks like we'll need to come up with a different way for candidates to submit their solutions.

In the meantime, I've updated the instructions for how to submit your solutions. Please find them in the README file of your assignment repository.

Please let me know if you have any other questions. And once again, my apologies for this delay.

I wish you the best of luck with the rest of your interview process.
